common/concurrent/ExecutorService.java

```diff
@@ -19,23 +19,27 @@
 package grakn.core.common.concurrent;
 
 import grakn.core.common.concurrent.actor.EventLoopGroup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.Executors;
 import java.util.concurrent.ForkJoinPool;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 
 public class ExecutorService {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ExecutorService.class);
```

Comment:
not used

Reply:
we're keeping them in every class for now, cause we intend to add improved logging soon